strength_percentiles.py

```diff
@@ -30,55 +32,65 @@
-        'amateurs': [],
-        'professionals': [],
-        'men': [],
-        'women': [],
-        'raw': [],
-        'equipped': []}
-
-def scrape_webpage_and_store_meet_results(quote_page):
-    page = urllib2.urlopen(quote_page)
-    soup = BeautifulSoup(page, 'html.parser')
-    meet_results_table = soup.find_all('tr') # all rows of meet results table,
-                                             # each corresponds to an
-                                             # individual's performance at the
-                                             # meet
-    for row in meet_results_table: # information for one lifter
-        parse_and_store_result(row)
+def parse_result_and_categories(row):
+    """
+    Parses table row of BeautifulSoup object for categories (gender,
+    professional status, equipment) and result (squat, bench, deadlift) of
+    the associated lifter.
 
-def parse_and_store_result(row):
+    Parameters:
+    -----------
+    row: bs4.element.tag
+        A table row of BeautifulSoup object which contains category and result
+        information of a lifter in the expected format (see SQUAT_COLUMN,
+        BENCH_COLUMN, etc.)
+
+    Returns:
+    --------
+    result, categories: Result is tuple of ints, or None, containing the
+    (squat, bench, deadlift) of the lifter. categories is tuple of strings,
+    containing the (gender, professional_status, equipment) of the
+    lifter.  Returns None if the row contains headers and no information.
+
+    """
     row = row.contents # list of HTML objects in row
     if row[1].get('colspan'): # row contains column headers and no data, ignore
         return
-    gender = get_data_from_table(row,GENDER_COLUMN) # string
-    professional_status = get_data_from_table(row,PROFESSIONAL_STATUS_COLUMN) # ...
-    equipment = get_data_from_table(row,EQUIPMENT_COLUMN) # ...
-    squat = get_data_from_table(row,SQUAT_COLUMN) # float
-    bench = get_data_from_table(row,BENCH_COLUMN) # ...
-    deadlift = get_data_from_table(row,DEADLIFT_COLUMN) # ...
+    gender = get_data_from_table(row, GENDER_COLUMN) # string
+    professional_status = get_data_from_table(row, PROFESSIONAL_STATUS_COLUMN) # ...
+    equipment = get_data_from_table(row, EQUIPMENT_COLUMN) # ...
+    squat = get_data_from_table(row, SQUAT_COLUMN) # float or None
+    bench = get_data_from_table(row, BENCH_COLUMN) # ...
+    deadlift = get_data_from_table(row, DEADLIFT_COLUMN) # ...
 
     if squat and bench and deadlift: # Valid entries for each lift in row
         total = squat + bench + deadlift
     else:
         total = None
 
-    result = (squat, bench, deadlift, total)
-    categories = (gender, professional_status, equipment)
-    categorize_and_store_result(categorized_results,result,categories)
+    result_and_categories = (gender, professional_status, equipment, squat,
+        bench, deadlift, total)
+    return result_and_categories
 
 def get_data_from_table(row, column):
```

Comment:
This function name is somewhat suspect - it really only grabs lift data right? If so, I'd recommend renaming it.